### PR TITLE
Create lol_US.lang

### PR DESCRIPTION
**Pull request by jtrent238**
LOLCAT Language

*** First 60 Lines of the Language file!!! ***

### CODE REVIEW

**Comment by Muirrum** [comment]
Um, is this recognized by Minecraft?

**Comment by jtrent238** [comment]
Yes, It is in the language menu as LOLCAT! _**ALL VERSIONS 1.9+**_

File: src/main/resources/assets/mo/lang/lol_US.lang [hunk 1]
```diff
@@ -0,0 +1,62 @@
+##########################################################################################################
+# Items
+
+item.matter_scanner.name=MATTR SCANNR
+item.emergency_ration.name=EMERGENCY RASHUN
+item.phaser.name=FASR
+item.battery.name=BATTERY
+item.creative_battery.name=CREATIV BATTERY
+item.me_conversion_matrix.name=MATTR-ENERGY CONVERSHUN MATRIX
+item.isolinear_circuit.mk1.name=ISOLINEAR CIRCUIT MK1
```
Comment (Muirrum) [inline src/main/resources/assets/mo/lang/lol_US.lang:10]: No new things for these?

File: src/main/resources/assets/mo/lang/lol_US.lang [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+item.isolinear_circuit.mk4.name=ISOLINEAR CIRCUIT MK4
+item.upgrade.base.name=UPGRADE SHELL
+item.h_compensator.name=HEISENBERG COMPENSATOR
+item.integration_matrix.name=INTEGRASHUN MATRIX
+item.s_magnet.name=Superconductor MAGNET
+item.matter_dust_refined.name=REFIND MATTR DUST
+item.matter_dust.name=MATTR DUST
+item.matter_dust.details=PUT IN RECYCLR 2 REFINE
+item.machine_casing.name=MACHINE CASIN
+item.dilithium_crystal.name=DILITHIUM CRYSTAL
```
Comment (Muirrum) [inline src/main/resources/assets/mo/lang/lol_US.lang:22]: No new name for this either?

File: src/main/resources/assets/mo/lang/lol_US.lang (hunk 1 above)
Comment (jtrent238) [reply src/main/resources/assets/mo/lang/lol_US.lang:10]: They are spelled slightly differently, and in all caps.

File: src/main/resources/assets/mo/lang/lol_US.lang (hunk 2 above)
Comment (jtrent238) [reply src/main/resources/assets/mo/lang/lol_US.lang:22]: They are spelled slightly differently, and in all caps.

File: src/main/resources/assets/mo/lang/lol_US.lang (hunk 1 above)
Comment (Muirrum) [reply src/main/resources/assets/mo/lang/lol_US.lang:10]: Not this one: item.isolinear_circuit.mk1.name=ISOLINEAR CIRCUIT MK1

File: src/main/resources/assets/mo/lang/lol_US.lang (hunk 2 above)
Comment (Muirrum) [reply src/main/resources/assets/mo/lang/lol_US.lang:22]: Not this one: item.dilithium_crystal.name=DILITHIUM CRYSTAL

Comment (jtrent238) [reply src/main/resources/assets/mo/lang/lol_US.lang:22]: Yea I can fix it later.

**Comment by Muirrum** [comment]
I mean, why not?

**Pull request merged**